test/state/host.cpp

```diff
@@ -276,7 +276,7 @@ evmc::Result Host::call(const evmc_message& orig_msg) noexcept
         return evmc::Result{EVMC_FAILURE, orig_msg.gas};  // Light exception.
 
     auto state_snapshot = m_state;
-    auto logs_snapshot = m_logs.size();
+    const auto logs_snapshot = m_logs.size();
```

Comment:
It look like refactoring not related to the commit.

Reply:
True, but I don't think this is worth a separate commit. The change is mostly stylistic.